### PR TITLE
Select only subset of reads from multi_fast5

### PR DESCRIPTION
**Pull request by lpryszcz**
Added optional retrieval for subset of reads (`-l` / `--limit_by_read_ids`) to `multi_to_single_fast5.py`. Also added `-r` so it can be used instead of `--recursive`.

### CODE REVIEW

**Comment by fbrennen** [comment]
Hi @lpryszcz -- thanks for the PR. Can you make this work the way it does in `conversion_tools.multi_fast5_subset`, supporting tsv files like `sequencing_summary.txt`?

**Comment by lpryszcz** [comment]
Note, this can be easily achieved using
`multi_to_single_fast5.py -r -i guppy -s reads -l <(tail -n+2 sequencing_summary.txt | cut -f2)`

**Comment by fbrennen** [comment]
Thanks very much -- supporting tsv files will make the tools consistent.

File: ont_fast5_api/conversion_tools/single_to_multi_fast5.py [hunk 1]
```diff
@@ -107,12 +111,14 @@ def main():
                         help="Number of reads per multi-read file")
     parser.add_argument('-t', '--threads', type=int, default=1, required=False,
                         help="Number of threads to use")
-    parser.add_argument('--recursive', action='store_true',
+    parser.add_argument('-r', '--recursive', action='store_true',
                         help="Search recursively through folders for for single_read fast5 files")
+    parser.add_argument('--revert', action='store_true',
```
Comment (fbrennen) [inline ont_fast5_api/conversion_tools/single_to_multi_fast5.py]: Could we separate this out into a second PR? We haven't really thought about adding picopore-style functionality into here.

Comment (lpryszcz) [reply ont_fast5_api/conversion_tools/single_to_multi_fast5.py]: yeap, sorry, didn't want to mess up the stuff. I'll try to sort this out this week. 
btw, I've implemented `--revert` as picopore isn't maintained any more. Beside, picopore's revert doesn't really work as advertised. And doing --revert on the fly on old fast5 during single_to_multi_fast5.py makes lots of sense as you spare lots of time and IO operations. Worked like charm for me :)

**Comment by fbrennen** [comment]
We've got [fast5_subset](https://github.com/nanoporetech/ont_fast5_api#fast5_subset) now which I think should let you do this.

**Pull request closed without merging**